tests/Avalonia.Base.UnitTests/AvaloniaObjectTests_Metadata.cs

```diff
@@ -65,13 +65,13 @@ private class Class1 : AvaloniaObject
                 AvaloniaProperty.Register<Class1, string>("Styled", "foo");
 
             public static readonly DirectProperty<Class1, string> DirectProperty =
-                AvaloniaProperty.RegisterDirect<Class1, string>("Styled", o => o.Direct, unsetValue: "foo");
```

Comment:
@grokys I think this changes the control flow of the test. Please, verify that it still actually checks for what it was intended to check

Reply:
I restored it, but I think there may be a DirectProperty design error. If in a derived class, where I have overwritten DirectProperty metadata, if I access through the CLR property I might have a value other than DirectProperty.

Reply:
`unsetValue` is intended as metadata, and it's the control author's responsibility to keep the metadata in sync with the code. This is because direct properties are just wrappers around standard CLR properties.